stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/nativeimpl/serviceendpoint/Stop.java

```diff
@@ -43,8 +43,9 @@
 )
 public class Stop extends AbstractGrpcNativeFunction {
 
-    public static void stop(Strand strand, ObjectValue serverEndpoint) {
+    public static Object stop(Strand strand, ObjectValue serverEndpoint) {
         getServerConnector(serverEndpoint).stop();
         serverEndpoint.addNativeData(HttpConstants.CONNECTOR_STARTED, false);
+        return null;
```

Comment:
If this is returning only null why do we have to return `Object` from the method? Doesn't the ballerina counterpart return `nil` only?

Reply:
The current implementation, we are not actually doing anything here. because graceful shutdown not supported yet. but I added this because in future we maybe need to send an error. So we can do it without any change in the method signature.